src/plugin.spec.ts

```diff
@@ -77,11 +77,11 @@ test('Wechaty Plugin uninstaller should be called after wechaty.stop()', async t
 
 test('isWechatyPluginUninstaller()', async t => {
   const FIXTURES = [
-    [undefined, false],
-    [() => {}, true],
+    [ undefined, false ],
```

Comment:
Please create another PR to fix the linting problem.

We will be able to merge that PR very quickly so that this PR will be clearer.

Thanks!